x-pack/plugins/cases/public/components/add_comment/index.tsx

```diff
@@ -79,6 +80,9 @@ export const AddComment = React.memo(
         schema,
       });
 
+      const appCapabilities = useKibanaCapabilities();
```

Comment:
Could you please move this logic in `x-pack/plugins/cases/public/components/markdown_editor/editor.tsx`? This way, every component that uses the markdown editor will have lens disabled if there are no permissions.

Reply:
I moved it there in the last commit.